models/marts/history_assets.sql

```diff
@@ -1,5 +1,6 @@
 {{ config(
-    materialized='incremental'
+    tags = ["history_assets", "trade_agg"]
```

Comment:
do we still use the `history_assets` table in the trade_agg pipeline? I thought you just refactored to remove

Reply:
Ah you're right. `history_assets` was still a CTE but it wasn't used.